wagtail_localize/views/submit_translations.py

```diff
@@ -57,9 +53,7 @@ def __init__(self, instance, *args, **kwargs):
 
         # Don't count page aliases as existing translations. We can convert aliases into properly translated pages
         if isinstance(instance, Page):
-            existing_translations = existing_translations.exclude(
```

Comment:
Sorry, I think I may have blacked this file with the wrong settings!

Reply:
suggestion: `pre-commit install && pre-commit run black --all-files`